analytics-datastore-elastic-search/swarm.sh

```diff
@@ -163,7 +157,7 @@ function destroy_package() {
     docker::try_remove_volume "$STACK" es-data
   fi
 
-  docker::prune_configs "elasticsearch"
+    docker::prune_configs "elasticsearch"
```

Comment:
fix indentation

Reply:
Pretty sure this is just an outdated commit that you looked at, since on my local there is no indentation issue and when trying to edit the file on Github itself it doesn't pickup any whitespace changes (and loads with correct indentation)